mkdocs.yml

```
@@ -85,6 +85,14 @@ nav:
             - 'Building manual': variants/talos_2/building-manual.md
             - 'Installation manual': variants/talos_2/installation-manual.md
             - 'Hardware configuration matrix': variants/talos_2/hardware-matrix.md
+        - 'Clevo NV41':
+            - 'Releases': variants/clevo_nv41/releases.md
+            - 'Building': variants/clevo_nv41/building.md
+            - 'EC firmware update': variants/clevo_nv41/ec_update.md
+            - 'Flashing with external programmer': variants/clevo_nv41/flashing_external.md
+            - 'Flashing with internal programmer': variants/clevo_nv41/flashing_internal.md
+            - 'Hardware Configuration Matrix' : variants/clevo_nv41/testing_configuration.md
```

some rows have a space before colon, some not. It seems that syntax should be without spaces
```suggestion
            - 'Hardware Configuration Matrix': variants/clevo_nv41/testing_configuration.md
```

Please apply to other rows as well

---

Fixed